engine/src/loaders/texture/png_loader.cpp

```diff
@@ -273,10 +269,6 @@ void PngLoader::handle4bppPalletized(TextureBuilderData* result,
   auto* pixel = static_cast<unsigned char*>(result->data);
   struct PngClut* clut = (struct PngClut*)result->clut;
 
-  for (int i = numPallete; i < 16; i++) {
-    memset(&clut[i], 0, sizeof(clut[i]));
-  }
-

```

Comment:
unnecessary function, this function already does it before:
memset(result->clut, 0, getTextureSize(8, 2, bpp32));